source/administration/config-settings.rst

```diff
@@ -221,6 +221,15 @@ If using HTTP (insecure), this is the maximum time allowed from the end of readi
 | This feature's ``config.json`` setting is ``"WriteTimeout": 300`` with numerical input. |
 +-----------------------------------------------------------------------------------------+
 
+Idle Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Set an explicit idle timeout in the HTTP server. This is the duration allowed before an idle connection is disconnected.
```

Comment:
Should we be consistent like the other timeouts by saying "This is the _maximum time_ allowed" ?

Reply:
Great catch. I've updated it. Thanks! :)